README.md

```diff
@@ -1,48 +1,54 @@
-# Bio-Formats
+# OME Common C++
 
-[![Build Status](https://travis-ci.org/openmicroscopy/bioformats.png)](http://travis-ci.org/openmicroscopy/bioformats)
+OME Common is a standalone C++ library required by other OME C++
+projects for common functionality which is not readily available from
+the C++ Standard Library.  This includes basic portability functions,
+to wrapping other libraries to make them usable with Modern C++
+programming practices.
 
-Bio-Formats is a standalone Java library for reading and writing life sciences
-image file formats. It is capable of parsing both pixels and metadata for a
-large number of formats, as well as writing to several formats.
+It serves a similar purpose to the OME Common Java library, with some
+shared functionality, though for the most part they are quite
+different.
```

Comment:
Since the OME Common Java library is not effective yet, I would wait we have a placeholder to cross-reference it.